dependencies.gradle

```diff
@@ -9,7 +9,7 @@ ext {
       project.getProperty('square.agpVersion') : '4.2.2'
 
   kotlinVersion = project.hasProperty('square.kotlinVersion') ?
-      project.getProperty('square.kotlinVersion') : '1.7.20'
+      project.getProperty('square.kotlinVersion') : '1.8.0-RC'
```

Comment:
Always a stable release should be the default.

Reply:
Same as other comment, this is temporary and will be changed back in the final changes